sphinx/config.py

```diff
@@ -125,6 +125,7 @@ class Config(object):
         primary_domain = ('py', 'env', [NoneType]),
         needs_sphinx = (None, None, string_classes),
         needs_extensions = ({}, None),
+        manpages_url_pattern = (None, 'env'),
```

Comment:
I guess the error comes from here. This is not renamed.

Reply:
fixed.